src/test/common/steps/common-steps.js

```diff
@@ -93,12 +95,24 @@ async function completeTheApplicationAsAWomanWhoIsNotPregnant () {
   await enterCardAddress(ADDRESS_LINE_1, ADDRESS_LINE_2, TOWN, POSTCODE)
 }
 
+async function setupWiremockMappings () {
```

Comment:
I would maybe have these in a new separate file as they are used in multiple places